javascript-forms/form-values.js

```diff
@@ -0,0 +1,16 @@
+var contactForm = document.querySelector('#contact-form');
+contactForm.addEventListener('submit', pushUserInput);
+
+function pushUserInput(event) {
+  event.preventDefault();
+  var name = contactForm.elements.name.value;
+  var email = contactForm.elements.email.value;
+  var msg = contactForm.elements.message.value;
+  var inputs = {
+    'Name:': name,
+    'Email:': email,
+    'Message:': msg
```

Comment:
These property names are capitalized _and_ include a colin in them, that's very confusing. I would just make these all lowercase and remove the colon from the property name